### PR TITLE
Feature/dashboard parents

### PR DESCRIPTION
**Pull request by mihaimazareanu**

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -17,12 +18,12 @@ export default function Home({
...
-  onSignIn,
-  users,
 }) {
+  const {user, setUser} = useContext(UserContext);
+  console.log(user);
   return (
     <>
-      {!signedIn ? (
+      {user === null || user.length === 0 ? (
         <>
```
Comment (actyralf) [inline pages/index.js:35]: `user` is now an object and objects don't have a `length` property. So please remove this check.

File: pages/index.js [hunk 2]
```diff
@@ -169,11 +200,46 @@ const ChildButton = styled.button`
...
+  width: 7rem;
+  box-shadow: 4px 4px 8px 1px rgba(166, 31, 43, 0.65);
+
+  :hover {
+    background-color: #a61f43;
+    box-shadow: 4px 4px 8px 1px rgba(166, 31, 67, 0.65);
+    transform: scale(1.1);
+  }
+`;
+
```
Comment (actyralf) [inline pages/index.js:241]: You have a styled component for every button and they share a lot of code. Please consider to use a single `StyledButton` component and configure it with props. (DRY=don't repeat yourself!)

**Review by actyralf — CHANGES_REQUESTED** [review]
Well done, Mihai!
Please have a look at my comments below...

**Review by actyralf — APPROVED** [review]
Great that you did the refactoring of your styled components!

PR is approved, BUT GitHub found a security issue: In an older version of your code you had visible MongoDB credentials (see message of the "Git Guardian Bot" on your pull request page). It's very difficult to remove these old occurrences from git history - so please CHANGE YOUR MONGODB credentials and make sure you have them ONLY in your .env file.
There are thousands of bots scanning Github for such credentials every day, so your database is now open for everyone.

**Comment by mihaimazareanu** [comment]
Thank you , Ralf, I'll take a look immediately at how to change the MONGODB Credentials. The comment from GitGuardian comes only because I have installed the GitGuardian App, to test it, now I disabled it, so it won't show any such messages anymore. 😄

**Pull request merged**